src/norm_oracles.jl

```diff
@@ -47,12 +47,14 @@ function compute_extreme_point(lmo::LpNormLMO{T,1}, direction; kwargs...) where
     return ScaledHotVector(-lmo.right_hand_side * sign_coeff, idx, length(direction))
 end
 
-function compute_extreme_point(lmo::LpNormLMO{T,p}, direction; kwargs...) where {T,p}
+function compute_extreme_point(lmo::LpNormLMO{T,p}, direction; v = similar(direction), kwargs...) where {T,p}
     # covers the case where the Inf or 1 is of another type
     if p == Inf
-        return compute_extreme_point(LpNormLMO{T,Inf}(lmo.right_hand_side), direction)
+        v = compute_extreme_point(LpNormLMO{T,Inf}(lmo.right_hand_side), direction)
```

Comment:
v should be passed as keyword here

Reply:
you mean `v = compute_extreme_point(LpNormLMO{T,Inf}(lmo.right_hand_side), direction, v=v)`?

Reply:
yes